src/kibana/plugins/vis_types/controls/vislib_basic_options.html

```diff
@@ -17,4 +17,28 @@
       Scale Y-Axis to Data Bounds
     </label>
   </div>
+  <div class="vis-option-item">
+    <label>
+      <input type="checkbox" ng-model="vis.params.setYExtents.max" ng-checked="vis.params.setYExtents.max">
+      Set Y-Axis Max Value
+    </label>
+    <div ng-show="vis.params.setYExtents.max">
+      <label>
+        y-max
+        <input type="number" step="0.1" ng-model="vis.params.yAxis.max" ng-required="vis.params.setYExtents.max">
```

Comment:
I don't actually won't to limit people from having values less than 0. You could have values where the min and the max values for the chart are both below zero.